en/complete_vehicles/px4_vision_kit.md

```diff
@@ -38,9 +38,14 @@ The guide explains the minimal additional setup required to get the vehicle read
    Collision prevention can be tested in position mode provided there is a good position lock from either GPS or optical flow.
 1. The port labeled `USB1` may jam the GPS if used with a *USB3* peripheral (disable GPS-dependent functionality including missions).
    This is why the boot image is supplied on a *USB2.0* memory stick.
-1. The *UP Core* should only be powered using the battery (do not remove the *UP Core* power socket safety cover).
+1. For PX4 Vision with ECN below 010 (carrier board RC04 and below), the *UP Core* should **only** be powered using the battery (do not remove the *UP Core* power socket safety cover).
```

Comment:
Consider moving this point to the end and turning it into a warning. 

So 

1. PX4 Vision with ECN 010 or above (carrier board RC05 and up), the *UP Core* can be powered by either the DC plug or with battery.

  ![RC Number](../../assets/hardware/px4_vision_devkit/RC.png)
  
:::warning
For PX4 Vision with ECN below 010 (carrier board RC04 and below), the *UP Core* should **only** be powered using the battery (do not remove the *UP Core* power socket safety cover).

 ![ECN Number](../../assets/hardware/px4_vision_devkit/Serial_Number_Update.jpg)
 ![Warning - do not connect power port](../../assets/hardware/px4_vision_devkit/warning_power_port_update.png)
:::